index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,100;1,300;1,500;1,600;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>SI - PortFolio</title>
+  </head>
+  <body>
+    <header>
+      <nav id="nav-bar" class="nav-bar">
+        <div class="logo" id="logo">
+          <h2>
+            <a href="#">Si<span class="logo-dot">.</span></a>
+          </h2>
+        </div>
+        <div class="nav-links">
+          <img class="menu-icon" src="images/Union.svg" alt="menu-icon" />
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <section class="hero-container">
+        <div class="hero-section">
+          <h2 class="intro-msg">I’m Sendy <br />Glad to see you!</h2>
```

Comment:
- [ ] Good job with the markup of your project Sendy. However, this `<h2>` should be a `<h1>` since it is the main heading of your page. Kindly change it to a `<h1>` instead of a `<h2>`  and it will be semantically correct.